app/actions/configurations.py

```diff
@@ -0,0 +1,10 @@
+from .core import PullActionConfiguration, AuthActionConfiguration
+
+
+class AuthenticateConfig(AuthActionConfiguration):
+    username: str
+    password: str
```

Comment:
We are using `pydantic.SecretStr` for password fields now. Since this integration may serve as an example for future devs, it'd be great to apply it here.
https://github.com/PADAS/gundi-integration-bluetrax/blob/0cf4eebefc648fc517a07633cbcfdfaa9d2f68ef/app/actions/configurations.py#L8